File: P33_1209/P33_1209.py
n = int(input())
k = int(input())
items = []
for i in range(n):
    items.append(int(input()))

sum = 0
for i in range(n):
    if(i == k):
        continue
    else:
        sum += items[i]
bill = sum/2
print(bill)

File: P33_1209/P33_1209.c
#include<stdio.h>

int main()
{
   int n,k;
   scanf("%d%d",&n,&k);
   int items[n];
   for(int i=0 ; i<n ; i++)
   {
       scanf("%d",&items[i]);
   }
   float sum=0;
   for(int i=0 ; i<n ; i++)
   {
       if(i == k)
       {
          continue;
       }
       else
       {
           sum += items[i];
       }
   }
   float bill;
   bill = sum/2;
   printf("%f",bill);
   return 0;
}